RocketChat/Rocket.Chat.ReactNative
# Chore: Hooks app/views/SelectServerView

## Proposed changes


## Issue(s)	


## How to test or reproduce

- Join 2 or more servers
- Then into the mobile's gallery select something to share by Rocket.Chat
- Click on server
- You should navigate to SelectServerView
- Change the server

## Screenshots

## Types of changes




## Checklist



## Further comments
